src/platform/brand-consolidation/containers/BrandConsolidation.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+function BrandConsolidation({ brandConsolidationEnabled, children }) {
+  if (brandConsolidationEnabled) {
+    return <div data-brand-consolidation-enabled>{children}</div>;
+  }
+  return null;
+}
+
+export function mapStateToProps(state) {
+  return {
+    brandConsolidationEnabled: state.buildSettings.brandConsolidationEnabled
```

Comment:
Where is the `buildSettings` prop coming from?

Reply:
There's a directory, `platform/monitoring/BuildSettings` that contains a reducer that moves that data into the store. The data itself is created from the `window.settings` property, from the `js/settings.js` file. The reason we did it this way is that eventually the content will have different build cycle than the app code, so we need a way to pass data from the content to the code. This was the easiest way I could think to do that.